### PR TITLE
refs #12 - Deployment HNZ

### PR DESCRIPTION
**Pull request by YmaIneo**

### CODE REVIEW

**Review by FlorentP42 — APPROVED** [review]

File: s104-n104-ubuntu2004/fledge/fledge.dockerfile
Comment (aklira) [inline s104-n104-ubuntu2004/fledge/fledge.dockerfile:1]: Version fledge_2.1.0-803 is not available anymore. Need to update to latest version.

**Review by aklira — CHANGES_REQUESTED** [review]

File: sHNZ-n104-ubuntu2004/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    container_name: fledge
+    image: fledgepower/fledge
+    build:
+      context: fledge/
+      dockerfile: fledge.dockerfile
+    ports:
+      - "8081:8081"
+      - "1995:1995"
+      - "6683:6683"
+      - "6001:6002"
```
Comment (FlorentP42) [inline sHNZ-n104-ubuntu2004/docker-compose.yml:15]:
```suggestion
      - "6001:6001"
      - "6002:6002"
```

File: sHNZ-n104-ubuntu2004/fledge/fledge.dockerfile [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+WORKDIR /usr/local/fledge
+
+COPY importModules.sh importModules.sh
+COPY start.sh start.sh
+
+RUN chmod +x start.sh
+VOLUME /usr/local/fledge 
+
+# Fledge API port for FELDGE API http and https and Code Server
+EXPOSE 8081 8090 1995 8080 2404 2405
```
Comment (FlorentP42) [inline sHNZ-n104-ubuntu2004/fledge/fledge.dockerfile:160]:
```suggestion
EXPOSE 8081 8090 1995 8080 2404 2405 6001 6002
```

File: sHNZ-n104-ubuntu2004/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+services:
+
+  fledge-power:
+    container_name: fledge
+    image: fledgepower/fledge
+    build:
+      context: fledge/
+      dockerfile: fledge.dockerfile
+    ports:
+      - "8081:8081"
```
Comment (FlorentP42) [inline sHNZ-n104-ubuntu2004/docker-compose.yml:12]: use 8090 here?

**Review by FlorentP42 — APPROVED** [review]

**Review by aklira — APPROVED** [review]

**Pull request merged**